Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -104,24 +104,6 @@ class OverviewViewController: UIViewController {
         self.recentActivityTableView.backgroundView = label
         self.recentActivityTableView.separatorStyle = .none
     }
-    
-    @IBAction func showAllTransactionsButtonTap(_ sender: Any) {
-        self.navigateToMenu(.history)
-    }
-    
-    @IBAction func showSendPage(_ sender: Any) {
-        self.navigateToMenu(.send)
-    }
-    
-    @IBAction func showReceivePage(_ sender: Any) {
-        self.navigateToMenu(.receive)
-    }
```

Comment:
The buttons referencing these functions still exist on the storyboard and still attempt to make these connections when clicked, which causes the app to crash. Please delete the buttons from the storyboard.

Reply:
resolved in 2c74553

Reply:
resolved in 2c74553